libs/opsqueue_python/pyproject.toml

```diff
@@ -60,7 +60,3 @@ test = [
 # We ensure tests never rely on global state,
 # by running them in a random order, and in parallel:
 addopts = "--random-order --workers=auto"
-# Individual tests should be very fast. They should never take multiple seconds
-# If after 20sec (accomodating for a toaster-like PC) there is no progress,
-# assume a deadlock
-timeout=20
```

Comment:
I guess the CI servers are sometimes a 'toaster-like PC'? :see_no_evil:
Are you sure you want to remove the deadlock protection? Maybe bumping this to, say, 60 seconds, would be enough?

Reply:
The plot thickens: Why does `pytest-timeout` not get loaded as part of the Nix overlay?

Reply:
it's not listed in default.nix like for example `pytest-parallel`. adding it there also seems to fix the issue